internal/engine/upper.go

```diff
@@ -150,6 +152,9 @@ func (u Upper) CreateManifests(ctx context.Context, manifests []model.Manifest,
 					event.manifest,
 					buildState)
 				if err != nil {
+					if isPermanentError(err) {
```

Comment:
does this error get surfaced to the user in a useful way?

Reply:
yes